src/api/databases/sql/models/FavoriteTheme.ts

```diff
@@ -11,7 +11,7 @@ class FavoriteTheme extends Model { }
 FavoriteTheme.init({}, { sequelize, modelName: "FavoriteTheme" });
 
 // contains only user id and theme id to associate user favorites
-FavoriteTheme.belongsTo(User, { foreignKey: "user_id" });
-FavoriteTheme.belongsTo(Theme, { foreignKey: "theme_id" });
+FavoriteTheme.belongsTo(User, { foreignKey: "userId" });
+FavoriteTheme.belongsTo(Theme, { foreignKey: "themeId" });
```

Comment:
This is a general question.

There are a bunch of field names change for the database tables. I do not see any migration scripts for them. Does it mean we do not have to worry about production data migration?

Reply:
The database will be altered by sequelize, so it's taken care of! 😊